docs/glossary.rst

```diff
@@ -0,0 +1,83 @@
+Glossary
+========
+
+.. glossary::
+   :sorted:
+
+   WebMon
+      Monitor for instrument status and autoreduction status at SNS and HFIR. WebMon can refer to
+      either the whole system or the landing page at https://monitor.sns.gov.
+
+   DASMON
+      Data Acquisition System Monitor (DASMON) is a per-beamline service that reports PV values and
+      experiment/run meta-data to WebMon. Due to the high volume of PV updates, DASMON writes PV
+      updates straight to the database. DASMON reports its status to WebMon for system diagnostics.
+
+   DAS
+      Data Acquisition System.
+
+   SMS
+   Stream Management Service
+      Service that aggregates data from fast neutron event data and slow PVs into a data stream
+      available for both live monitoring and file archiving. SMS reports its status to WebMon for
+      system diagnostics.
+
```

Comment:
Your formatting makes it look like a double entry (IMO).  I suggest keeping the definition all together
SMS
Stream Management Service (SMS) - Service that aggregates data from fast neutron event data and slow PVs into a data stream available for both live monitoring and file archiving. SMS reports its status to WebMon for system diagnostics.